mdsuite/__init__.py

```diff
@@ -43,6 +44,7 @@
     Report.__name__,
     "config",
     Molecule.__name__,
+    "units",
```

Comment:
can it be `units.__name__`?

Reply:
In this case it can't. `units` is the module and not the class. I don't think modules come with a `__name__`. We could add it just like with `__version__` but I rather keep it like this.

Reply:
I thought it still worked actually but it is all good in any case